### DISCUSSION TITLE
Can I change the informative icon of a neutral Callout component?

### DISCUSSION THREAD

**Discussion by Danii2020** [twilio-labs/paste#3267] [Q&A] [answered]
Hey Twilio Paste team, I was wondering if it possible to change the informative icon of the Callout component on its neutral variant, I want to use the `Support` icon instead to have something like the image below but without the informative icon:
<img width="1498" alt="image" src="https://github.com/twilio-labs/paste/assets/69876673/3d01abc6-2c7e-4959-b874-7cc2396f0b2b">

My other alternative is to build a component from scratch but I don't want to do that since the Neutral Callout component already has the styles I need with the exception of the icon.

Thanks in advance.

**Comment by nkrantz** [comment, marked as answer]
Hi @Danii2020 - unfortunately the Callout component doesn't allow you to swap out other icons. If you're committed to that design and you decide to build your own, feel free to take a look at [our Callout code](https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/callout/src/Callout.tsx) and copy it. Otherwise, another option might be creating a Popover with a "**Need some help?**" Button, [like we have on our website](https://paste.twilio.design/components/callout#neutral-callout) (above the table of contents on the right side of the page).

**Reply by Danii2020** [reply]
I get it, thanks for the suggestions!

**Reply by nkrantz** [reply]
@Danii2020 - one more thought. If you'd like to fill out our [Dependency Request](https://airtable.com/shrvMvL3BG6Zm0f9l) form requesting a Support variant of Callout, that would help us prioritize the work. Thanks!
